src/test/java/org/jboss/tm/usertx/JndiProvider.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2013, Red Hat, Inc., and individual contributors
```

Comment:
Some of the tests have been copied over from the narayana repo. When we have proper CI  testing of jboss-transaction-spi and a way to validating that narayana PRs don't break the SPI I will remove them from the narayana repo.